services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/SwitchFeature.java

```diff
@@ -26,5 +26,6 @@
     PKTPS_FLAG,
     MATCH_UDP_PORT,
     MAX_BURST_COEFFICIENT_LIMITATION,
-    MULTI_TABLE
+    MULTI_TABLE,
+    INACCURATE_SET_VLAN_VID_ACTION
```

Comment:
Are we sure that a switch performs "inaccurate set" operation? Or it's rather "inproper data" returned on the dump operation?

Also, what is exactly "inaccurate" in the vlan returned? You're still able to obtain the exact vlan value from OFOxmVlanVid object.

Reply:
It is inaccurate from OF spec perspective (maybe inaccurate is not best term here, propose another if you have some in mind). Command sent to the switch set "presence" bit in this field, but read operation do not return this bit. I suspect that switch completely ignores this "VLAN presence bit". We can perfome more test later when this issue will not affect our functionality.

Reply:
Ok. Please, create an issue for this.